sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 4.4.0-beta.2 (Unreleased)
+## 4.4.0 (Unreleased)
 
 ### Features Added
 
+- Added `KeyVaultSettingsClient` to get and update Managed HSM settings.
+
```

Comment:
Empty sections should be removed. See previous comment about the prepare-release script.